pkg/diagnose/connectivity/inventoryendpoint_test.go

```diff
@@ -397,27 +396,6 @@ func TestCheckGet(t *testing.T) {
 	assert.Contains(t, err.Error(), "invalid status code: 401")
 }
 
-func TestCheckGetConnectionFailure(t *testing.T) {
-	// Create a resolvedEndpoint with an invalid URL
-	endpoint := resolvedEndpoint{
-		url:    "http://invalid-url-that-does-not-exist.com",
```

Comment:
Not necessary for this PR, but if you want to add test for this in the future, you could use `invalid.com` as recommended by [RFC 2606](https://www.rfc-editor.org/rfc/rfc2606.html#section-2). Sorry for not catching this in the first review.